zipline/src/hostMain/kotlin/app/cash/zipline/EventListener.kt

```diff
@@ -72,6 +72,21 @@ abstract class EventListener {
     return null
   }
 
+  /**
+   * Invoked when an application load was skipped because the code is unchanged.
```

Comment:
Because the code is unchanged? Or because the code is stale?

Reply:
ah it's because it's stale!